### ISSUE TITLE
Issues With FBX Export from Blender 3.0

### ISSUE THREAD

**Issue by eArticleSolutions** [pKrime/Expy-Kit#12]
Funny thing, and I just want to confirm.

Today, Mixamo is rejecting all FBX files, which is odd since a few weeks ago, they were working. 

It is accepting OBJ, without armature data, which takes me to their auto-rigger.  The OBJ file is imported to mixamo, and when I get the new mocap (FBX), and import to Blender, the result is a very, very, very tiny little model.

Older FBX files that I created with Blender 2.9 work very well.

Here is a screenshot attached.

Kindly help.

Thanks

![fbx_not_working](https://user-images.githubusercontent.com/17984836/152602060-b7da3003-f942-436f-9744-5234b6f2fa63.jpg)

**Comment by eArticleSolutions** [comment]
here is the rig. I have deleted everything else and cleared all orphan data. I have made the rig game friendly and converted the bone names. Kindly let me know what step to take next

[please_check.zip](https://github.com/pKrime/Expy-Kit/files/8006278/please_check.zip)
.

**Comment by pKrime** [comment]
Hi, I have just exported the armature from your .blend without doing anything, and I can see the nicely animated skeleton in mixamo. Perhaps there is something in the mesh that mixamo doesn't like, it can be the materials or even something in the polygon/object count. If you confirm that exporting the armature in please_check.blend works, I would try to export a simpler mesh with no materials.

regards,
p.

**Comment by eArticleSolutions** [comment]
I will try to remove all materials and see if it works.

**Comment by eArticleSolutions** [comment]
I used the armature alone as you did, and downloaded it as an fbx with skin, and it worked when I imported it back in blender. Since all I need is the mocap data, I think I will work with it this way. I think the issue is my mesh, which generated a very high poly count when the subdivisions were applied during fbx export.

Thanks for your help. You have really saved my skin.

Keep up the wonderful work.
